packages/react-reconciler/src/ReactFiber.new.js

```diff
@@ -488,6 +490,10 @@ export function createFiberFromTypeAndProps(
           expirationTime,
           key,
         );
+      case REACT_DEBUG_TRACE_MODE_TYPE:
```

Comment:
Any way to make this branch DEV-only, and fall through to unknown case otherwise?

Reply:
Hm... well... we could fork the entire `switch` statement, but that doesn't seem good.

We could maybe...also pre-filter type, e.g.
```
let filteredType = type;
if (!__DEV__ && type === REACT_DEBUG_TRACE_MODE_TYPE) {
  filteredType = null;
}
```

But neither of those really feel like improvements.